features/landing-page/Navbar.tsx

```diff
@@ -0,0 +1,75 @@
+import { AuthForm } from "features/auth/AuthForm"
+import { useState } from "react"
+import { useTranslation } from "next-i18next";
+
+export default function Navbar() {
```

Comment:
1. default exports are a bad practice and shouldn't be used (nextjs pages are a a exception)
2. please use `const = () =>` instead of `function`, it's generally a convention to do so in react codebases